backend/src/federation/index.ts

```diff
@@ -28,7 +41,9 @@ export const startDHT = async (
   try {
     const TOPIC = DHT.hash(Buffer.from(topic))
 
-    const keyPair = DHT.keyPair()
+    const keyPair = DHT.keyPair(getSeed())
```

Comment:
I think you can omit the function and just write
```suggestion
    const keyPair = DHT.keyPair(Buffer.alloc(CONFIG.FEDERATE_DHT_SEED, ' hex' ))
```

You might require to check length of the secret tho - consider to use the config for this (?) or do the check before

```suggestion
    if(CONFIG.FEDERATE_DHT_SEED.length < DHT.minKeyLength){
      throw new Error()
    }
    const keyPair = DHT.keyPair(...)
```

Reply:
found a solution to fix the buffer to the required size of 32 independent from input
i kept the getSeed(), because the Buffer.alloc doesn't handle a set or not set property to return null for the DHT.keyPair invokation.